roles/ansible/tower/config-ansible-tower-ocp-ldap-ca/tasks/ocp_set_ca_certificate.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Check for CA ConfigMap
+  command: |
+    oc get configmap ldap-pem \
+      -n {{ openshift_project }}
+  register: ldap_ca_check
+  failed_when: ldap_ca_check.rc > 1
+
+- name: Create and Mount CA cert as configmap
+  block:
+    - name: Create configmap for CA
+      command: |
+        oc create configmap ldap-pem \
+          --from-file={{ ansible_tower.ldap.ca_cert }} \
+          -n {{ openshift_project }}
+    - name: Mount configmap as volume in /etc/certs direcotry
+      command: |
+        oc set volume deployment/ansible-tower \
+          --add \
+          --configmap-name ldap-pem \
+          --type configmap \
+          --mount-path /etc/certs/{{ ansible_tower.ldap.ca_cert | basename }} \
```

Comment:
this mount path needs to match what's in the `ldap.py` file - e.g.: right now hardcoded to `/etc/certs/ldap.pem`